package.json

```diff
@@ -21,7 +21,7 @@
   },
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/huston007/file-selector.git"
```

Comment:
This is not pointing to the proper git url 😉

Reply:
Ouch, my bad. Did copy the url from fork page without even thinking.
Thanks!